demo/page/component/Badge.vue

```diff
@@ -0,0 +1,24 @@
+<template lang="html">
+  <div>
+    <Demo title="Simple badge">
+      <h4>Header <p-badge>5</p-badge></h4>
```

Comment:
Can you add multiple examples of badges colors please? Like it was was done on [getpapercss.com](https://www.getpapercss.com/#badges)

Reply:
#27